helper.go

```diff
@@ -32,6 +32,16 @@ type queryable interface {
 	QueryRow(string, ...interface{}) *sql.Row
 }
 
+// BatchSplitter is an interface with method which returns byte slice for
+// splitting SQL batches. This need to split sql statements and run its
+// separately.
+//
+// For Microsoft SQL Server batch splitter is "GO". For details see
+// https://docs.microsoft.com/en-us/sql/t-sql/language-elements/sql-server-utilities-statements-go
+type BatchSplitter interface {
+	Splitter() []byte
```

Comment:
What about moving the interface and method to lower case? I don't think this need to be exported